backend/src/main/java/ai/verta/modeldb/utils/JdbiUtil.java

```diff
@@ -0,0 +1,104 @@
+package ai.verta.modeldb.utils;
+
+import ai.verta.modeldb.common.CommonConstants;
+import ai.verta.modeldb.common.CommonDBUtil;
+import ai.verta.modeldb.common.CommonMessages;
+import ai.verta.modeldb.common.MssqlMigrationUtil;
+import ai.verta.modeldb.config.MDBConfig;
+import io.grpc.health.v1.HealthCheckResponse;
+import java.io.File;
+import java.sql.SQLException;
+import java.util.concurrent.Executor;
+import liquibase.exception.LiquibaseException;
+import liquibase.resource.FileSystemResourceAccessor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.hibernate.exception.JDBCConnectionException;
+
+public class JdbiUtil extends CommonDBUtil {
+  private static final Logger LOGGER = LogManager.getLogger(JdbiUtil.class);
+  private static JdbiUtil jdbiUtil;
```

Comment:
why is this a static thing, rather than having an instance that can just be injected like normal?

Reply:
Removed this file from this PR. i will take this in new PR